pkg/reader/ccip.go

```diff
@@ -775,6 +775,81 @@ func (r *ccipChainReader) GetContractAddress(contractName string, chain cciptype
 	return addressBytes, nil
 }
 
+// GetLinkPriceUSD gets the LINK price in USD from the FeeQuoter contract on the destination chain.
```

Comment:
Maybe state the denomination here (i.e how many decimals)

Reply:
Aside: these implicit decimals/denominations are very confusing imo